src/core/types/function/date-global.d.ts

```diff
@@ -0,0 +1,7 @@
+export {};
+
+declare global {
+  interface Window {
+    Date: typeof Date;
+  }
+}
```

Comment:
Should this be in this directory? Could it be scoped to the root dir?

Reply:
@rcebulko and I are in the middle of bike-shedding what to do about the `Window` globals. Since he's out I followed the pattern he started (of scoping as closely as possible).

My preference would be to group together all of the builtins into a single `window.d.ts` file.

Reply:
I think you're right, my approach made more sense when using reference tag but since we can't we may as well combine